test/js/pageobjects/DocumentUpload.js

```diff
@@ -0,0 +1,18 @@
+import Base from './BasePage.js'
+
+class DocumentUpload extends Base{
+    getUploadInput() { return (async ()=>{
+      const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
+      this.driver.executeScript(function(el) {
+        el.setAttribute('style','display: block !important')
+      },input)
+      return input
+    })()}
+
+    getUploadButton() { return this.$('.onfido-sdk-ui-Uploader-button')}
+
+    copy = (lang="en") =>
```

Comment:
I asked it here initially: https://github.com/onfido/onfido-sdk-ui/pull/629/files#r279306351

From a JS newbie:

> I was going to ask why do we need to copy (i assumed it was because we were mutating the instance) but it seems my mistake was probably the name.
> Are you sure you mean copy? Isn't this more like a fetcher? Because in fact we aren't copying right? We are always using the same instance?

Reply:
sorry, by copy I mean text. it's not a very familiar/common term outside the UK though

Reply:
renamed to strings